test/test_external.py

```diff
@@ -514,3 +514,11 @@ def test_use_api_name_in_call_method():
     # app = gr.load(name="spaces/gradio/multiple-api-name-test")
     # assert app(15, api_name="minus_one") == 14
     # assert app(4, api_name="double") == 8
+
+
+def test_load_custom_component():
+    demo = gr.load("spaces/freddyaboulton/gradiopdf")
```

Comment:
This test is currently passing on CI because we import gradio_pdf.PDF in a separate test. But it fails if you run it standalone. Would be good to import `gradio_pdf` here so it passes on its own
```suggestion
    from gradio_pdf import PDF
    demo = gr.load("spaces/freddyaboulton/gradiopdf")
```

Reply:
Good call